static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -268,4 +268,9 @@
         }
     }
 
+    .media-primary & {
+        @include mq($from: mobile, $until: mobileLandscape) {
+            margin-top: -1rem;
```

Comment:
I see two options which might still be better. 

1. Add something here https://github.com/guardian/frontend/blob/master/common/app/views/support/HtmlCleaner.scala to change the class

2. There is so much nesting in this file which we really try to avoid. I am hesitant to allow any more until this is refactored to be without nesting. As a solution for this problem, instead of relying on the  `media-primary` class, you can add a class to the atom with a `--main-media` modifier that has the `margin-top: -1rem;`